packages/fluentui/e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,97 @@
+const CYPRESS_ACTION_TIMEOUT = 10 * 1000;
+
+Cypress.Commands.add('gotoTestCase', (testFilePath, waitForSelector) => {
+  const { _ } = Cypress;
+  const path = testFilePath
+    .split('/')
+    .pop()
+    .replace(/^(.+).spec.ts?$/, '$1')
+    .replace(/^\//, '');
+
+  cy.visit(`/${_.kebabCase(path)}`);
+  cy.exist(waitForSelector);
+});
+
+Cypress.Commands.add('exist', selector => {
+  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('exist');
+});
+
+Cypress.Commands.add('notExist', selector => {
+  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('not.exist');
+});
+
+Cypress.Commands.add('nothingIsFocused', () => {
+  cy.focused().should('not.exist');
+});
+
+Cypress.Commands.add('simulatePageMove', () => {
+  cy.get('body').realTouch();
+});
+
+Cypress.Commands.add('nothingIsFocused', () => {
+  cy.focused().should('not.exist');
+});
+
+Cypress.Commands.add('clickOn', selector => {
+  cy.get(selector).realClick();
+});
+
+Cypress.Commands.add('focusOn', selector => {
+  cy.get(selector).focus();
+});
```

Comment:
I would like to get rid of custom commands as it will be confusing for people who are not familiar with our codebase, _but we can do it in a follow-up PR_.